### DISCUSSION TITLE
Remote for a SkyQ box

### DISCUSSION THREAD

**Discussion by surveyor2** [GerryDazoo/Slinger#187] [General]
Does anyone have a custom remote for a SkyQ box? In particular the EXIT code 36 in the standard remote doesn't get me out of the Guide or Menu once I'm in there. None of the existing code buttons do.

Thanks in advance

James

**Comment by AlanRichey** [comment]
http://www.hifi-remote.com/forums/dload.php?action=file&file_id=13978&sid=1ae710c021dd6f11096f0ed289b67ce3

Al

**Reply by surveyor2** [reply]
Hi Alan,
I remember downloading them back in the day but I need the uncompiled version so I can see the codes. Are they there somewhere?

Jim

**Reply by AlanRichey** [reply]
Here is the RemoteMaster file I used to build it.    I've had to rename the extension to .txt to get it through so if you need to use it in RemoreMaster you need to rename is as .rmdu

[Sky Q Box.txt](https://github.com/GerryDazoo/Slinger/files/9890081/Sky.Q.Box.txt)

Although if you just open it as a text file you can read the codes direct

**Reply by surveyor2** [reply]
Thanks very much

**Reply by surveyor2** [reply]
The exit code is 21, by the way.

15 (hex) in the RemoteMaster source file needs to be changed to 21 (decimal) in the Slinger remote.txt

James

**Comment by GerryDazoo** [comment]
Have a look at the release notes on how to spelunk for working keycodes. In a nutshell. The server when it starts a stream will print out all the valid remote keys for the configured source.
In the "Channel" field of the web remote put in ? followed by the keycode you want to test and see what happens.

**Comment by GerryDazoo** [comment]
I went to that link you posted and only saw the .bin files. Where did you get the .rndu?

**Reply by AlanRichey** [reply]
I built it :-)   If you go to my earlier reply to surveyor2, download the file and change the extension to .rmdu you will have the file.

Al

**Comment by GerryDazoo** [comment]
So is there a repository for the RemoteMaster files? people could use those to figure out the keycodes needed by my remote stuff instead of my cumbersome trial and error method. It looks like it would also be pretty easy to parse and write a little script to auto-generate the needed remote file for my code. I need one form my Motorola 3400M. :-)

**Reply by GerryDazoo** [reply]
Found them!

**Reply by AlanRichey** [reply]
Just in case anyone else needs them:  http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=98

**Reply by RandomMan719** [reply]
So I got a .bin file from hifi-remote.com.  I loaded it in a custom remote, through Slingplayer2.0, and it works correctly.  How do we make them work with Slinger?  Is there an easy way to convert them, so Slinger can use them?

**Reply by RandomMan719** [reply]
Can Remote Master convert them for Slinger?
